sanity/package.json

```diff
@@ -15,10 +15,14 @@
     "sanity"
   ],
   "dependencies": {
+    "@heroicons/react": "^2.1.5",
```

Comment:
I would recommend you to not use any external icons packages

Reply:
I'm not using it in Next.js tho, just in Sanity. I think it provides a good service, like setting an icon for social media type. Will this hurt that much, especially when used only in Sanity Studio?